dev/com.ibm.ws.cdi.internal/resources/com/ibm/ws/cdi/internal/resources/CDI.nlsprops

```diff
@@ -108,3 +108,7 @@ unversioned.beansXML.warning.CWOWB1018W.useraction=Update the beans.xml file wit
 no.bda.for.module.CWOWB1019E=CWOWB1019E: WebSphere Application Server internal error occurred. Contact WebSphere Application Server support with the following data: {0} {1}
 no.bda.for.module.CWOWB1019E.explanation=CMSG0001
 no.bda.for.module.CWOWB1019E.useraction=See the problem determination information on the WebSphere Application Server Support web page: http://www.ibm.com/software/webservers/appserv/was/support.
+
+dev.mode.enabled.CWOWB1020W=CWOWB1020W: Weld developer mode has been removed
```

Comment:
Change
Weld developer mode has been removed
to
Weld development mode has been removed.

(I am not familiar with this feature, but the proper term for it seems to be Weld development mode, with Weld capitalized, and "development" instead of "developer" or "dev": https://docs.jboss.org/weld/reference/2.3.0.Final/en-US/html/devmode.html)